ChurchCRM/CRM
# Bugfix/calendar issues

#### What's this PR do?
Reverts react-bootstrap to use Bootstrap 3 instead of 4.
Fixes the data type of the `Date` passed to the calendar's event modal dialog

#### What Issues does it Close?

Closes #5292
Closes #5291 
Closes #5287 

### Steps to test:

*  Open the calendar view in 4.1.0, and click on either an existing event, or click in a "blank area" to create a new event
*  Observe the Javascript console indicates `RangeError: Invalid time value`, and the dialog does not appear.

*  Now, checkout this branch and run `npm install` and `npm run build-react`, then refresh the calendar page and observe that the steps above result in the modal dialog being displayed.
